src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -6,18 +6,35 @@ import * as ssn from '../../common/schemaform/definitions/ssn';
 
 import { relationshipLabels, genderLabels } from '../utils/helpers';
 
-const defaultProps = [
-  'relativeFullName',
-  'relativeSocialSecurityNumber',
-  'relativeDateOfBirth',
-  'gender',
-  'relationship'
-];
+const defaults = {
+  fields: [
+    'relativeFullName',
+    'relativeSocialSecurityNumber',
+    'relativeDateOfBirth',
+    'gender',
+    'relationship'
+  ],
+  required: [
+    'relativeFullName',
+    'relativeSocialSecurityNumber',
+    'relativeDateOfBirth',
+    'relationship'
+  ],
+  labels: {}
+};
+
+
+/**
+ * Returns an applicantInformation page based on the options passed.
+ *
+ * @param {Object} schema   The full schema for the form
+ * @param {Object} options  Options to override the defaults above
+ */
+export default function applicantInformation(schema, options) {
```

Comment:
I think we should use this as the preferred way of passing options to the pages because it mitigates need to copy a default parameter if a later parameter needs to be overridden and it's cleaner than specifying a bunch of default parameters in-line.

Thoughts?